### DISCUSSION TITLE
Monetization: Paid perk / Subscription / Status etc

### DISCUSSION THREAD

**Discussion by alltheseas** [dskvr/nostr-watch#663] [Ideas]
**Is your feature request related to a problem? Please describe.**
In the long run, if nostr succeeds, nostr.watch *should* have a way to monetize independent of grant sources.

**Describe the solution you'd like**
I am not sure what it is, but nostr.watch should have a paid feature, status, perk etc. My gut feel is that a freemium model is the way.

It could be that paid relays:
1) can afford to share some sats flow with nostr.watch
2) would be interested in having premium positioning on nostr.watch listings

I suppose some may call this the ad model. 

**Describe alternatives you've considered**
Rely on the goodwill of @dskvr and Opensats forever.

**Additional context**
What are other ways might we monetize nostr.watch?

**Comment by alltheseas** [comment]
![image](https://github.com/sandwichfarm/nostr-watch/assets/64376233/b9dc2144-0957-4c73-b1e8-83d95b9d9407)

**Comment by dskvr** [comment]
Definitely something I've thought about over the last year.

1. Higher Resolution Testing
2. Higher Resolution Uptime Monitoring with Notifications
3. Sponsorship Levels
